include/Foundation/NSObjCRuntime.h

```diff
@@ -47,6 +47,11 @@
 
 #if !defined(NS_NONATOMIC_IOSONLY)
 #define NS_NONATOMIC_IOSONLY nonatomic
```

Comment:
You should remove this line then

Reply:
Sorry about that! I thought I got rid of that line. Apparently not...